OpenEXR_CTL/CMakeLists.txt

```
@@ -12,8 +12,8 @@ add_subdirectory( exr_ctl_exr )
 add_custom_target( OpenEXR_CTL DEPENDS CtlModules exrdpx exr_ctl_exr )
 
 if ( PKG_CONFIG_FOUND )
-configure_file(../config/OpenEXR_CTL.pc.in "${PROJECT_BINARY_DIR}/OpenEXR_CTL.pc" @ONLY)
-install( FILES "${PROJECT_BINARY_DIR}/OpenEXR_CTL.pc" DESTINATION lib/pkgconfig COMPONENT dev )
+    configure_file(../config/OpenEXR_CTL.pc.in "${PROJECT_BINARY_DIR}/OpenEXR_CTL.pc" @ONLY)
+    install( FILES "${PROJECT_BINARY_DIR}/OpenEXR_CTL.pc" DESTINATION lib/pkgconfig COMPONENT dev )
```

```suggestion
    install( FILES "${PROJECT_BINARY_DIR}/OpenEXR_CTL.pc" DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig COMPONENT dev )
```

Looks like you missed to use the GNUInstallDirs variable here.